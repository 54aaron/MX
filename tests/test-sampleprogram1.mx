int main() {
    Matrix m;
    Matrix n;

    m = [[2,4],[6,8],[10,12]];
    n = m';
    print_matrix(m);
    print_matrix(n);
    return 0;
}
