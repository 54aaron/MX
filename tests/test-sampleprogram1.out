[ 2, 4 ]
[ 6, 8 ]
[ 10, 12 ]

[ 2, 6, 10 ]
[ 4, 8, 12 ]

